main.go

```diff
@@ -77,6 +80,7 @@ func init() {
 	flag.BoolVar(&saveOutput, "O", false, "Save body as remote filename")
 	flag.StringVar(&outputFile, "o", "", "output file for body")
 	flag.BoolVar(&showVersion, "v", false, "print version number")
+	flag.StringVar(&clientCertFile, "c", "", "client cert file for tls config")
```

Comment:
What do you think about using `E` which is the same flag `cURL` uses? This project is a Go version of `httpstat.py`, which is just a wrapper around `cURL`. So we should try to keep as much parity as possible.